cv-tech/src/SchoolsList.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+const schools = require('./schools.json');
+
+
+
+class SchoolsList extends Component {
+
+	handleClick(src) {
+		console.log(src);
```

Comment:
Apparently- the furthest anyone got on this interview question was logging this out.